SPECS/intel-ice-alt.spec

```diff
@@ -14,14 +14,15 @@
 
 Summary: %{vendor_name} %{driver_name} device drivers
 Name: %{vendor_label}-%{driver_name}-alt
-Version: 1.8.8
+Version: 1.11.17.1
 Release: 1%{?dist}
 License: GPLv2
 
 # Extracted from latest XS driver disk
-Source0: intel-ice-1.8.8.tar.gz
-
-Patch1: 0001-Look-for-firmware-in-lib-firmware-override.patch
+Source0: intel-ice-1.11.17.1.tar.gz
+Patch0: fix-enabling-sr-iov-with-xen.patch
+Patch1: 0001-CP-47698-Change-module-name-intel_auxiliary-to-intel.patch
+Patch2: 0001-Look-for-firmware-in-lib-firmware-override.patch
```

Comment:
Since this patch is not from XS and we now have other patches coming from XS, let's start putting it under a `# XCP-ng patches` comment and start with number 1000.